mappings/net/minecraft/entity/mob/AbstractPiglinEntity.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_5418 net/minecraft/entity/mob/AbstractPiglinEntity
+	FIELD field_25758 immuneToZombification Lnet/minecraft/class_2940;
```

Comment:
```suggestion
	FIELD field_25758 IMMUNE_TO_ZOMBIFICATION Lnet/minecraft/class_2940;
```

Reply:
There is still a field with that name in `PiglinEntity`, because :thonkjang: forgot to remove it. I would also rename the field in the subclass to `OLD_IMMUNE_TO_ZOMBIFICATION`, with a javadoc comment saying how it's one that Mojang forgot to remove (see the one in `ZombieEntity`).
Also while you're at it, `PiglinEntity.dancing` needs capitalizing.